docker-compose.dev.yml

```diff
@@ -19,7 +19,9 @@ services:
       - '27017-27019:27017-27019'
 
   dtfs-central-api:
-    build: ./dtfs-central-api
+    build:
+      context: .
+      dockerfile: ./dtfs-central-api/Dockerfile
```

Comment:
This change is made so that we can build from the root and take into account files from the common library.

This will not result in code from other unrelated packages be added to the docker container, as the Dockerfile in each package only copies across its own package, plus the libs directory which contains the common package.